theme/static/style/style.css

```diff
@@ -440,6 +487,39 @@ pre code {
         padding-left: 1.8rem;
         text-align: left;
     }
+
+    /*features page*/
+    .features .w50 {
+        padding-top: 2rem;
+    }
+    .features .col-8 {
+        min-width: 100%;
+        max-width: 100%;
+    }
+    .features .col-3 {
+        min-width: 100%;
+        max-width: 100%;
+    }
+    .features img {
+        min-width: 95%;
+        max-width: 95%;
```

Comment:
Again, use width 100% and add padding and border-box properties to control spacing between elements. I think it is best if you comment out all styles for responsive and start working on each element one by one until you are satisfied with it.

Reply:
box-sizing: border-box doesn't work for me, the images go out of the screen due to the margin, that is why I left the margin at 1.8rem and lowered the width to 95%.